libs/orders/src/lib/components/order-actions-dropdown/order-actions-dropdown.tsx

```diff
@@ -7,14 +7,21 @@ import {
   VegaIconNames,
 } from '@vegaprotocol/ui-toolkit';
 import { t } from '@vegaprotocol/i18n';
+import { useRef } from 'react';
 
 export const OrderActionsDropdown = ({ id }: { id: string }) => {
+  const ref = useRef<HTMLButtonElement>(null);
   return (
     <DropdownMenu
+      onOpenChange={(open) => {
+        if (open) ref.current?.classList.add('open');
+        if (!open) ref.current?.classList.remove('open');
+      }}
       trigger={
         <DropdownMenuTrigger
-          className="hover:bg-vega-light-200 dark:hover:bg-vega-dark-200 p-0.5 focus:rounded-full hover:rounded-full"
+          className="hover:bg-vega-light-200 dark:hover:bg-vega-dark-200 [&.open]:bg-vega-light-200 dark:[&.open]:bg-vega-dark-200 p-0.5 rounded-full"
```

Comment:
This repeats in many places. I suggest to make component from it `<ActionMenuTrigger />`